config/locales/en.yml

```diff
@@ -851,6 +851,7 @@ Please follow the instructions there to make your enterprise visible on the Open
   registration_detail_state_error: "State required"
   registration_detail_country: "Country:"
   registration_detail_country_error: "Please select a country"
+  registration_detail_state_label: "abbr"
```

Comment:
@KeirOsborn This `registration_detail_state_label` key doesn't seem to be used anywhere now...? Was it for a translation that got removed?